components/molecules/BenefitCard.js

```diff
@@ -77,9 +91,36 @@ export function BenefitCard(props) {
             </p>
             <span className="text-sm font-semibold pl-1">{t("youMayBe")}</span>
 
-            <ReactMarkdown className="prose md:max-w-none mt-1">
+            <ReactMarkdown
+              className={`prose md:max-w-none mt-1 ${
+                expanded === true ? "line-clamp-3" : "line-clamp-none"
```

Comment:
Sorry for being nit-picky but "expanded === true" can be simplified with "expanded" since it will always be equal to true or false.

(This is repeated 4 times in the file.)

Reply:
This is what Tom added.